easybuild/easyconfigs/d/dm-haiku/dm-haiku-0.0.11-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,57 @@
+# update 0.9.11: Thomas Hoffmann (EMBL) 
+easyblock = 'PythonBundle'
+
+name = 'dm-haiku'
+version = '0.0.11'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/deepmind/dm-haiku'
+description = """Haiku is a simple neural network library for JAX developed by some of the authors of Sonnet, a neural
+network library for TensorFlow."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('jax', '0.4.25', versionsuffix),  # required by jmp, also provides absl-py
+    ('PyYAML', '6.0'),
+    ('CUDA', '12.1.1', '', SYSTEM),
+    ('tensorstore', '0.1.53'),
+    ('protobuf-python', '4.24.0'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('jmp', '0.0.4', {
+        'checksums': ['5dfeb0fd7c7a9f72a70fff0aab9d0cbfae32a809c02f4037ff3485ceb33e1730'],
+    }),
+    ('flax', '0.8.1', {
+        'checksums': ['ce3d99e9b4c0d2e4d9fc28bc56cced8ba953adfd695aabd24f096b4c8a7e2f92'],
+    }),
+    ('optax', '0.1.9', {
+        'checksums': ['731f43e8b404f50a5ef025b1261894d7d0300f7ad9cb688ea08f67b40822e94f'],
+    }),
+    ('orbax_checkpoint', '0.5.3', {
+        'modulename': 'orbax',
+        'checksums': ['1572904cbbfe8513927e0d80f80b730e0ef2f680332d3c2810d8443532938b45'],
+    }),
+    ('toolz', '0.12.1', {
+        'checksums': ['ecca342664893f177a13dac0e6b41cbd8ac25a358e5f215316d43e2100224f4d'],
+    }),
+    ('chex', '0.1.85', {
+        'checksums': ['a27cfe87119d6e1fe24ccc1438a59195e6dc1d6e0e10099fcf618c3f64771faf'],
+    }),
+    ('nest_asyncio', '1.6.0', {
```

Comment:
This belong before orbax_checkpoint since it is a dependency of orbax_checkpoint